docs/source/technical_tutorials/working_with_r/access_aws_maap.ipynb

```diff
@@ -0,0 +1,1577 @@
+{
```

Comment:
<p>For consistency, either put curly braces around <code>x$Key</code> like you did in a previous cell, or <em>remove</em> the braces in the previous cell. I recommend removing the earlier braces, rather than adding them here. Either way, just use consistent syntax.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/471/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>